db/migrate/20180315001307_change_tag_list_not_empty.rb

```diff
@@ -0,0 +1,10 @@
+class ChangeTagListNotEmpty < ActiveRecord::Migration[5.1]
+  def change
+    Image.all.each do |image|
+      next unless image.tag_list.empty?
+      image.tag_list = '<unset>'
+      image.save
+      # Image.destroy(image)
```

Comment:
1. Should the commented out code be removed?
2. `.save` will fail quietly if sth. goes wrong.
3. In migration, we should avoid using ActiveRecord and prefer plain SQL. `.save` will trigger the ActiveRecord validations. At this point, it's safe to do this. But what if in the future, you add more validations to the model?

Reply:
Makes sense.  We'll update.